analysed-packages/pyserial/version-3.5/pyserial-3.5-SPDX2TV.spdx

```diff
@@ -0,0 +1,1125 @@
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/documentation/examples.rst
+SPDXID: SPDXRef-item462775
+FileChecksum: SHA1: 57fdf3ca6349e519935a6e6eecb392e70241e257
+FileChecksum: SHA256: e0a36acecb3ec75fc008dfe4a24d70e229e0508a436908812507de2dacf0b64f
+FileChecksum: MD5: 98053211c2db8a4d91aa7623416d9560
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/documentation/appendix.rst
+SPDXID: SPDXRef-item462776
+FileChecksum: SHA1: 243413a5830352642a21be19a1c6cef7308311e0
+FileChecksum: SHA256: aa60a9fbf0aed456b8fad60863428e55b3746c48fc4aa549fbb6c71a7b9d5280
+FileChecksum: MD5: 5042fb1fc806d08545de37c69b616d25
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: LicenseRef-BSD-3-Clause
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright (c) 2001-2020 Chris Liechti <cliechti@gmx.net> </text>
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/documentation/index.rst
+SPDXID: SPDXRef-item462777
+FileChecksum: SHA1: 4167e4d401e80faaf4a029189279bf97565743f4
+FileChecksum: SHA256: 63cca24b163e52c62149d40e465d164d1e95c87edcf639842446b3785ee741b9
+FileChecksum: MD5: 72a43c5136293067227781718f636a0d
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/documentation/shortintro.rst
+SPDXID: SPDXRef-item462778
+FileChecksum: SHA1: ff84226f243fbc4ceaa754126d6460f4ef8109ff
+FileChecksum: SHA256: 52f4891459132337212b88d6a04f712b73fe6a3c43d2de5b9fe774c675ed6454
+FileChecksum: MD5: 219a22180377d922c96a98127118522f
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/documentation/conf.py
+SPDXID: SPDXRef-item462779
+FileChecksum: SHA1: 3f3d79dc5305ad7220cc6b4e5fa8cca4a20bb3f4
+FileChecksum: SHA256: ff35c8edac7726d46fd4d802118e39a287c9481fd1a311e76ff3755b1bde8a25
+FileChecksum: MD5: 213d7babb5cac736d02ef770bc0e4c3d
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: pyserial-3.5.tar.gz/pyserial-3.5.tar/pyserial-3.5/PKG-INFO
+SPDXID: SPDXRef-item462781
+FileChecksum: SHA1: 79dffd25e52d273106498dc76489e5c1d800910e
+FileChecksum: SHA256: 29670d8dab451ad667b6fb20e177c332632cb40561748a9f6eb97f5d9ec5bab0
+FileChecksum: MD5: aa9f9cc9cd26887c64ff05d3891a387e
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: NOASSERTION
+
+
+
+##-------------------------
+## License Information
+##-------------------------
+
+
+  
+LicenseID: LicenseRef-BSD-3-Clause
+LicenseName: BSD 3-Clause "New" or "Revised" License
+ExtractedText: <text> Copyright (c) <year> <owner>. 
```

Comment:
Please remove copyright notice. After that regenerate json, rdf.xml and yaml files

Reply:
```suggestion
ExtractedText:
```

Reply:
done in Fossology server